includes/definitions.inc.php

```diff
@@ -1534,13 +1543,6 @@
 $config['os'][$os]['over'][0]['graph'] = 'device_toner';
 $config['os'][$os]['over'][0]['text']  = 'Toner';
 
-$os = 'richoh';
```

Comment:
I think that is a wrong definition? 
i don't see anything related to os 'richoh' discovery ... and there is already and 'ricoh' before it